backend/cfg/cfg_with_layout.mli

```diff
@@ -50,18 +50,33 @@ val remove_block : t -> Label.t -> unit
 val is_trap_handler : t -> Label.t -> bool
 
 val save_as_dot :
-  t ->
   ?show_instr:bool ->
   ?show_exn:bool ->
-  ?annotate_block:(Label.t -> string) ->
-  ?annotate_succ:(Label.t -> Label.t -> string) ->
+  ?annotate_instr:
```

Comment:
Wouldn't it be slightly easier to have two
parameters (_e.g._ `~annotate_basic` and
`~annotate_terminator`) rather than a list?
(another occurrence below)

Reply:
We still need the list because that's how we allow to have multiple cells in the table with annotations.
We could have 
```
  annotate_basic:(Format.formatter ->
    Cfg.basic Cfg.instruction ->
    unit)
    list ->
  annotate_terminator:(Format.formatter ->
    Cfg.terminator Cfg.instruction ->
    unit)
    list ->
```
But then we still would have to assert that the length is the same, otherwise the table would formatted in unpredictable/unlogical way.

Reply:
You are right, that is probably not better with two arguments.